src/controllers/UserRequests.js

```diff
@@ -47,4 +49,46 @@ export function sendApi() {
   //empty the textboxes (maybe only do this if the submission fails do to an error or no transcripts or something else?)
   document.getElementById("apiKey").value = "";
   document.getElementById("versionID").value = "";
+}
+
+
+export async function getBestIntents(){
+
+  // apparently this has to be initialized to a certain type, or else an error is caused.
```

Comment:
This is a little informal, perhaps you would want to do more research and reword this comment so that someone can understand why the error happens without the code.